pkg/config/validation/validation_test.go

```diff
@@ -5580,10 +5580,53 @@ func TestValidateSidecar(t *testing.T) {
 				},
 			},
 		}, false},
+		{"skip(1.11): sidecar egress only one wildcarded", &networking.Sidecar{
+			Egress: []*networking.IstioEgressListener{
+				{
+					Hosts: []string{
+						"*/*",
+						"test/a.com",
+					},
+				},
+			},
+		}, false},
+		{"skip(1.11): sidecar egress wildcarded ns", &networking.Sidecar{
+			Egress: []*networking.IstioEgressListener{
+				{
+					Hosts: []string{
+						"*/b.com",
+						"test/a.com",
+					},
+				},
+			},
+		}, true},
+		{"skip(1.11): sidecar egress duplicated with wildcarded same namespace", &networking.Sidecar{
+			Egress: []*networking.IstioEgressListener{
+				{
+					Hosts: []string{
+						"test/*",
+						"test/a.com",
+					},
+				},
+			},
+		}, false},
+		{"skip(1.11): sidecar egress duplicated with wildcarded same namespace .", &networking.Sidecar{
```

Comment:
what does "skip(1.11)" mean?

Reply:
Oh I see. We should test the warnings as well, using checkValidationMessage like some other tests